esphomeyaml/components/ota.rst

```diff
@@ -6,6 +6,15 @@ firmware binaries to your node without having to use an USB cable for
 uploads. esphomeyaml natively supports this through its ``run`` and
 ``upload`` helper scripts.
 
+.. note::
+  Please be aware that the ESP module must be reset after a serial 
```

Comment:
Only required on an ESP8266 AFAIK, please update to something like `that ESP8266 modules`

Reply:
No worries, I just changed that part.